test/step-function.localstack.test.ts

```diff
@@ -214,4 +214,61 @@ localstackTestSuite("sfnStack", (testResource, _stack, _app) => {
     },
     "hello world"
   );
+
+  test(
+    "call lambda $SFN wait",
+    (parent) => {
+      return new StepFunction(parent, "sfn2", async () => {
+        $SFN.waitFor(1);
```

Comment:
Should waitFor return a promise?

Reply:
What would it mean to not wait on a blocking statement? I saw it as `Wait` waits synchronously. It also a good way to show integrations can be both async and sync.